scripts/pkg/build_templates/opensearch/opensearch.rpm.spec

```diff
@@ -10,15 +10,17 @@
 %define data_dir %{_sharedstatedir}/%{name}
 %define log_dir %{_localstatedir}/log/%{name}
 %define pid_dir %{_localstatedir}/run/%{name}
+%{!?_version: %define _version 0.0.0 }
+%{!?_architecture_alt: %define _architecture_alt x86_64 }
```

Comment:
Should this be just `_architecture`?

Reply:
I want to be consistent between our python code and here.
It is not the version we are using, but alternative ones.

Reply:
Probably will make it simpler by using architecture.